odxtools/diaglayer.py

```diff
@@ -350,7 +350,7 @@ def global_negative_responses(self) -> NamedItemList[Response]:
         return self._global_negative_responses
 
     @property
-    @deprecated(details="use diag_data_dictionary_spec.tables")
+    @deprecated(details="use diag_data_dictionary_spec.tables")  # type: ignore[misc]
     def tables(self) -> NamedItemList[Table]:
```

Comment:
why?

Reply:
our version of `@deprecated` decorator is untyped, so we get
```
odxtools/diaglayer.py:353: error: Untyped decorator makes function "tables" untyped  [misc]
```

(`deprecation` is the second external module which provides that decorator, I don't want to switch this every other year...)